posts.go

```diff
@@ -1380,12 +1380,14 @@ Are you sure it was ever here?`,
 			IsFound        bool
 			IsAdmin        bool
 			CanInvite      bool
+			Hostname       string
```

Comment:
The app configuration is passed into every page, so we don't need this extra property. We can just use `.Host` in the templates.

Reply:
Oh cool, I can fix it but won't be until later today.

Reply:
Cool, no worries!